### DISCUSSION TITLE
Does diesel-async support deadpool 10 ?

### DISCUSSION THREAD

**Discussion by 1g0rkR** [weiznich/diesel_async#143] [General]
Hi there, I am trying to use PoolConfig to manually configure some points and put it to builder.
```
Pool::builder(manager)
        .config(pool_config)
        .runtime(deadpool::Runtime::Tokio1)
        .build()?
```
It fails to `pool_config`: "^^^^^^^^^^^ expected `deadpool::managed::config::PoolConfig`, found `PoolConfig`".
And it works with deadpool 9.5.

**Comment by weiznich** [comment]
Support for deadpool 0.10 is only available on the master branch of diesel-async yet.

**Reply by 1g0rkR** [reply]
Got it.

**Reply by lsunsi** [reply]
What I coincidence, I'm also interested in this version! Maybe I'll point to master as well
